automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/WorkspaceMethodConfigDetailsPage.scala

```diff
@@ -158,7 +158,7 @@ class LaunchAnalysisModal(implicit webDriver: WebDriver) extends OKCancelModal("
 
   def searchAndSelectEntity(entityId: String): Unit = {
     entityTable.filter(entityId)
-    Link(entityId + "-link").doClick()
+    Link(entityId + "-link" inside entityTable).doClick()
```

Comment:
prevent link not found

https://fc-jenkins.dsp-techops.broadinstitute.org/job/firecloud-ui-fiab-test-runner/3164/